src/controllers/users.controller.js

```diff
@@ -250,9 +250,17 @@ class UserController {
    * @returns {object} responses
    */
   async fetchAllUsers(req, res) {
-    const allUsers = await db.user.findAll({
-      attributes: { exclude: ['password'] }
-    });
+    const { role } = req.query;
+
+    const options = {
+      attributes: { exclude: ['password', 'receiveNotification', 'lastLogin', 'isVerified'] },
+    };
+
+    if (role) {
```

Comment:
Since you only want to include the `where` key in `options` if there is a `role`, you can re-write options like so
```
const options = {
  attributes: {
    // attributes go in here
  },
   ...role !== undefined && { role }
}
```
Check out this article that explains conditionally including properties in an object [here](https://medium.com/@mikeh91/conditionally-adding-keys-to-javascript-objects-using-spread-operators-and-short-circuit-evaluation-acf157488ede)

Reply:
Great, Let me try it out!

Reply:
This is awesome, Done!